<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreeXSimClass</class>
 <widget class="QWidget" name="FreeXSimClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreeXSimWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="sceneLbl">
     <property name="text">
      <string>XXX作战</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="genrateBtn">
       <property name="text">
        <string>生成数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startSimBtn">
       <property name="text">
        <string>开始仿真</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseBtn">
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="text">
        <string>停止仿真</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="increSpeedBtn">
       <property name="text">
        <string>加速</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="descSpeedBtn">
       <property name="text">
        <string>减速</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="curBsLbl">
     <property name="text">
      <string>当前倍速:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="elapseTimeLbl">
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="curSimPrbar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="residTimeLbl">
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FreeXSim.qrc"/>
 </resources>
 <connections/>
</ui>
